Trap Quest.inform/Inform/Extensions/Player/Exposure Addiction.i7x

```diff
@@ -0,0 +1,51 @@
+Exposure Addiction by Player begins here.
+
+no-panties permanent fetish is a number that varies.  no-panties permanent fetish is 0.
+Definition: yourself is fighting against no-panties:
+	if no-panties permanent fetish is 1 or the diaper addiction of the player > 15 or there is a worn no-panties tattoo or there is a worn daddy's little girl tattoo or the humiliation of the player >= 40000:
+		let C be a random worn knickers;
+		if C is diaper, decide no;
+		if C is knickers and C is not no protection, decide yes;
+	decide no.
+no panties reminder is a number that varies.  no panties reminder is 0.
+A time based rule (this is the fighting against no-panties fetish rule):
+	if the player is fighting against no-panties:
+		if no panties reminder <= 0:
+			let C be a random worn knickers;
+			if C is knickers, say "[bold type]You [one of][or]still [stopping]feel extremely uncomfortable in your [printed name of C][bold type]!  [one of]You can feel your body desperately wishing to be free of them[if diaper lover > 0] because you only want to wear diapers[end if], and it's affecting your reflexes and making it harder to think properly.[or][stopping][roman type][line break]";
+			now no panties reminder is 30;
+		decrease no panties reminder by 1.
+Report wearing knickers:
+	if the player is fighting against no-panties, say "You immediately feel extremely uncomfortable in these, thanks to your [if diaper lover >= 1]diaper fetish[otherwise]fetish for not wearing panties[end if].".
+This is the knickers can't be summoned with no panties fetish rule:
+	if summoning is 1 and (no-panties permanent fetish is 1 or the class of the player is faerie):
+		unless wearing-target is diaper or (wearing-target is no protection and diaper lover <= 0), rule fails.
+The knickers can't be summoned with no panties fetish rule is listed in the knickers wearability rules.
+
+
+topless permanent fetish is a number that varies.  topless permanent fetish is 0.
+Definition: yourself is fighting against topless:
+	if topless permanent fetish is 1 or there is a worn topless tattoo or the humiliation of the player >= 35000:
+		let C be a random worn bra;
```

Comment:
This is a pretty good implementation but personally I want to do it differently and also take into account some other things.

I would do it like this:

`Definition: yourself is fighting against topless:
    if topless permanent fetish is 1 or there is a worn topless tattoo or the humiliation of the player >= 35000:
        repeat with C running through worn breast covering clothing:
            if C is actually dense and C is top-in-place and C is not fully exposing and C is nipple covering, decide yes;
    decide no.`

I haven't tested that code to make sure all the terms are correct but you get the idea.  Also I think with the use of `nipple covering`, the `not fully exposing` becomes redundant.  But we can keep it in just in case.

Anyway, we now also need to change the "braless reminder" to make sure that it always warns the players that their breasts are covered.  We can do that with changing line 40 to `let C be a random worn actually dense top-in-place nipple covering clothing;` and the start of line 41 to `if C is clothing`.

I think it should also work to change line 44 to `Report wearing actually dense nipple covering clothing:`